tests/layer_tests/py_frontend_tests/test_torch_frontend.py

```diff
@@ -464,6 +464,85 @@ def forward(self, x):
         "Parameter", "Sin", "Tan", "Add", "Result"]
 
 
+def test_inlined_extension():
+    import openvino as ov
+    import numpy as np
+
+    rng = np.random.default_rng(42)
+
+    @ov.inlined_extension
+    def numpy_cos(x):
+        # numpy is not captured by tracing, so we use a custom extension
+        return torch.from_numpy(np.cos(x.numpy(force=True)))
```

Comment:
I think we can detect automatically if `torch.from_numpy` is used on execution path when you trace.
For example, you can patch `torch.from_numpy` with warning messages so user will see what places need patching